nym-vpn-apple/README.md

```diff
@@ -11,26 +11,22 @@ Welcome to the home of [NymVPN](https://nymvpn.com/en) iOS + iPadOS + macOS. The
 ```
 ### Dependancy requirements:
 ```
-brew, go, swiftlint, swift-protobuf, grpc-swift
+brew, go, swiftlint, swift-protobuf, grpc-swift, fastlane
```

Comment:
You omitted to include the main requirement here: an Apple machine 🫥

Reply:
I thought the folder name explains that and xcode project :D No probs, will add that